python/lsst/meas/extensions/piff/piffPsfDeterminer.py

```diff
@@ -109,6 +111,16 @@ class PiffPsfDeterminerConfig(BasePsfDeterminerTask.ConfigClass):
         doc="Include star images used for fitting in PSF model object.",
         default=False
     )
+    useCoordinates = pexConfig.ChoiceField[str](
+        doc="Which spatial coordinates to regress against in PSF modeling.",
+        dtype=str,
```

Comment:
`dtype` here is redundant (and not recommended) if you used the typo annotation above.

Reply:
Thanks, I didn't know that.  Will remove.